update/snapshot.go

```diff
@@ -0,0 +1,19 @@
+package update
+
+import "github.com/spf13/cobra"
+
+// used by CI to inject url for downloading with tatcli update.
+// value of urlUpdate injected at build time
+// full URL update is constructed with architecture var :
+// urlUpdate + "tatcli-" + architecture, tatcli is the binary
+var urlUpdateSnapshot string
```

Comment:
As there is no Github URL for ``urlUpdateSnapshot``, I'd rather not expose this feature, at least not in the default build to avoid user frustration.